www/about/policies/terms-of-service.md

```diff
@@ -97,23 +98,24 @@ payments is solely between those parties.
 
 1. Terms for Givers
 
-  1. When you make a payment to a Project, it is a payment to fund the Work
-     described in the Project's profile. You understand and acknowledge that
-your payment does not entitle you personally to receive any goods or services
-from the Project, unless the Project explicitly offers those things to Givers.
+  1. When you make a payment to a Project Owner, it is a payment to fund the
+     Work described in the Project's profile. You understand and acknowledge
+that your payment does not entitle you personally to receive any goods or
+services from the Project Owner, unless the Project Owner explicitly offers
```

Comment:
I would remove the clause starting "unless" and just leave that to the next section ("Project Owners are responsible...") so that expectations around goods/services/the Work are handled in just one place (unless there is a subtle distinction between Work and goods/services that I'm missing).